src/Entity/Task.php

```diff
@@ -1192,4 +1194,13 @@ public function setPrefetchedPackagesAndWeight($prefetchedPackagesAndWeight)
 
         return $this;
     }
+
+    public static function fixTimeWindow(Task $task)
```

Comment:
can we change the name? it is hard to tell what this function is doing

`presetDoneAfter` ?